src/components/MeasureList.tsx

```diff
@@ -68,6 +75,17 @@ export default function MeasureList(props: { measureList: Measure[] }) {
                       <td tw="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
                         {measure.revisionNumber}
                       </td>
+                      <td tw="px-6 py-4 whitespace-nowrap text-right text-sm font-medium">
+                        <button
```

Comment:
So, we are not using buttons from the component library

Reply:
I used it for CreateNewMeasure Component, but this measureList Component is created with no reusable components, So I didn't change the existing implementation.